### PR TITLE
Feat/fujia button

### LINKED ISSUE #3: To support unit tests with vitest
- added unit tests configs with vitest

### PR DESCRIPTION
**Pull request by fushenguang**
- added unit tests configs;
- added eslint configs;
- added git commit configs with husky;
- added basic unit tests for Button component.

closes #3, #2

### CODE REVIEW

File: src/SFCButton/index.vue [hunk 1]
```diff
@@ -4,6 +4,6 @@
 
 <script lang="ts">
 export default {
-  name: 'SFCButton',
+  name: "SFCButton",
```
Comment (fushenguang) [inline src/SFCButton/index.vue:7]: Is needless  this file?

@fushenguang

File: src/index.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // import { createApp } from 'vue';
 import { createApp } from 'vue/dist/vue.esm-browser';
 
-import TButton from './button';
+import TButton from './button/Button';
 // import SFCButton from './SFCButton/index.vue';
 // import JSXButton from './JSXButton';
```
Comment (fushenguang) [inline src/index.ts:6]: To remove redundant comments!

@fushenguang

**Review by fushenguang — COMMENTED** [review]
That's okay! No major problems.

File: src/SFCButton/index.vue (hunk 1 above)
Comment (fushenguang) [reply src/SFCButton/index.vue:7]: It'll be deleted later.

File: src/index.ts (hunk 2 above)
Comment (fushenguang) [reply src/index.ts:6]: Ok

**Review by DeandreFu — APPROVED** [review]
No major problems.

**Pull request merged**